earth_enterprise/src/common/khxml/khdom.h

```diff
@@ -864,6 +897,24 @@ FromElementWithChildName(khxml::DOMElement *elem,
   }
 }
 
+template <class T>
+void
+FromElementWithChildName(khxml::DOMElement *elem,
+                         const std::string &childTagName,
+                         MTVector<T> &vec)
+{
+  vec.clear();
```

Comment:
Technically there are race conditions here because another thread could be adding stuff to the vector after you clear it. You'll always end up with a valid vector but it may not be what you expect after calling this function. Given what our code does I don't think that will be a problem, but I thought I'd bring it up.